charts/harvester-pcidevices-controller/Chart.yaml

```diff
@@ -15,7 +15,7 @@ type: application
 # This is the chart version. This version number should be incremented each time you make changes
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
-version: 0.0.0-dev
+version: 0.1.0-dev
```

Comment:
pcidevices is at version 0.2.0, is this a different version? And if so, should I update the pcidevices repo to reflect this?

Reply:
this is just a baseline chart.. the actual versions are immaterial. the next PR into `release` will need to valid build version.